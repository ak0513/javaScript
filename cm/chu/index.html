<!DOCTYPE html>
<html lang="ko">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    	<!-- <meta property="og:url" content="공유시 이동 url"> -->
	<meta property="og:title" content="Close Your Eyes">
	<meta property="og:type" content="website">
	<!-- <meta property="og:image" content="./img/og-img.png"> -->
	<!-- <meta property="og:description" content="공유시 보여질 설명"> -->
    <!-- <link rel="icon" href="./img/favicon.ico"> -->
	<link rel="stylesheet" href="./css/docSlider.css">
    <link rel="stylesheet" href="./css/swiper.min.css">
	<link rel="stylesheet" href="./css/common.css">
	<link rel="stylesheet" href="./css/style.css">
	<script src="./js/docSlider.min.js"></script>
    <script src="./js/swiper.min.js"></script>
	<script src="./js/common.js"></script>
    <title>Close Your Eyes</title>
</head>
<body>
    <!-- skipNav -->
    <div id="skipNav">
        <a href="#container" class="skipNav-link">본문 바로가기</a>
    </div>
    <!-- //skipNav -->

    <div id="wrap">
        <!-- header -->
        <header id="header">
            <h1 class="h-tit1">CLOSE YOUR EYES</h1>
            <button type="button" class="btn-nav-open" aria-label="전체메뉴 열기" onclick="popOpen('#sideMenu', this);"><span></span></button>
            <!-- PC 로그인 전 -->
            <div class="header-wallet">Connect Wallet</div>
            <!-- //PC 로그인 전 -->
            <!-- PC 로그인 후 -->
            <div class="header-wallet login">0x43E8...0532</div>
            <!-- //PC 로그인 후 -->

            <ul class="header-sns">
                <li class="discord"><a href="https://discord.com/invite/cye" target="_blank"><span class="a11y">디스코드</span></a></li>
                <li class="twitter">
                    <a href="https://twitter.com/closeyoureyesV0" target="_blank">
                        <span class="a11y">트위터</span>
                        <span class="sns-txt">V0</span>
                   </a>
                </li>
                <li class="twitter">
                    <a href="https://twitter.com/CYE_nft" target="_blank">
                        <span class="a11y">트위터</span>
                        <span class="sns-txt">V1</span>
                    </a>
                </li>
                <li class="telegram"><a href="" target="_blank"><span class="a11y">텔레그램</span></a></li>
            </ul>
        </header>
        <!-- //header -->

        <!-- container -->
        <main id="container">
            <div class="docSlider">
                <!-- section01 -->
                <section class="section01">
                    <video id="video" loop muted autoplay controls>
                        <source src="./img/chuchuCAT_V3_promo.mp4" type="video/mp4">
                        이 문장은 여러분의 브라우저가 video 태그를 지원하지 않을 때 화면에 표시됩니다.
                    </video>
                </section>
                <!-- // section01 -->

                <!-- section02 -->
                <section class="section02">
                    <ul class="tiket">
                        <li class="tiket-item item1"><a href="https://opensea.io/collection/cye-chuchucat-v0" target="_blank" class="tiket-link"><span class="a11y">OPEN SEA</span></a></li>
                        <li class="tiket-item item2"><a href="https://twitter.com/closeyoureyesV0" target="_blank" class="tiket-link"><span class="a11y">V.O TWITTER</span></a></li>
                        <li class="tiket-item item3"><a href="https://twitter.com/CYE_nft" target="_blank" class="tiket-link"><span class="a11y">V.1 TWITTER</span></a></li>
                        <li class="tiket-item item4"><a href="https://open.kakao.com/o/gtG51ZUd" target="_blank" class="tiket-link"><span class="a11y">KAKAO TALK</span></a></li>
                        <li class="tiket-item item5"><a href="https://discord.com/invite/cye" target="_blank" class="tiket-link"><span class="a11y">DISCORD</span></a></li>
                    </ul>
                    <div class="tiket-bg"><span></span></div>
                </section>
                <!-- //section02 -->

                <!-- section03 -->
                <section class="section03">
                    <div class="team-wrap">
                        <h2 class="title">Project <span>Team</span></h2>
                        <ul class="team">
                            <li class="team-item item1">
                                <div class="team-img"><img src="./img/team1.png" alt=""></div>
                                <div class="team-info">
                                    <h3 class="team-title"><span>artist</span> 205</h3>
                                    <p class="team-text">
                                        "이공오" 작가는 국내 최고 아트토이 그룹 "NWT"의 대표 입니다.<br>
                                        현재 팝마트, 나이키, 텐센트 등 다양한 글로벌 브랜드와 협업 중이고, <br class="pc">일러스트레이터를 겸하며 새로운 IP 생산에 매진 중 입니다.<br>
                                        대표작으로는 astroCAT, rabbiTT, ohFACE, spaceHOOD, chuchuCAT 등이 있습니다.
                                    </p>
                                    <div class="team-sns">
                                        <a href="https://www.instagram.com/labo_205/" class="team-sns-link instagram" target="_blank">
                                            <span class="a11y">인스타그램</span>labo_205
                                        </a>
                                    </div>
                                </div>
                            </li>
                            <li class="team-item item2">
                                <div class="team-img"><img src="./img/team2.png" alt=""></div>
                                <div class="team-info">
                                    <h3 class="team-title"><span>Developer</span> ELIXIR</h3>
                                    <p class="team-text">
                                        만렙 개발자. 팝캣티움이라는 귀여운 고양이 밈 블록체인을 개발했습니다.<br>
                                        최근까지 CxNxD(클네다), Mimon(미몽), Paper Children(기부 NFT 프로젝트) 등 NFT 프로젝트를 성공적으로 개발했습니다.
                                    </p>
                                    <div class="team-sns">
                                        <a href="https://twitter.com/elixirevo" class="team-sns-link twitter" target="_blank">
                                            <span class="a11y">트위터</span>ELIXIR
                                        </a>
                                    </div>
                                </div>
                            </li>
                            <li class="team-item item3">
                                <div class="team-img"><img src="./img/team3.png" alt=""></div>
                                <div class="team-info">
                                    <h3 class="team-title">Marketing & CM Unit</h3>
                                    <p class="team-text">
                                        NFT 크리에이티브 유닛 - CYE의 방향성 기획, 커뮤니티 빌드업 & 운영, <br class="pc">이벤트 기획 및 운영 등 프로젝트 전반의 활동을 담당합니다.<br>
                                        PM - Rara C, PL - ZΞRO, Unit - Vella, 1ETH, LangYi, Raon, JUDDY
                                    </p>
                                </div>
                            </li>
                            <li class="team-item item4">
                                <div class="team-img"><img src="./img/team4.png" alt=""></div>
                                <div class="team-info">
                                    <h3 class="team-title"><span>Management</span> CAMP Jackson</h3>
                                    <p class="team-text">NFT 크리에이티브 그룹이자, 진성 홀더들로 구성된 팀으로 투자, 마케팅, 프로젝트 운영에 노하우가 축적된 사람들 입니다.</p>
                                </div>
                            </li>
                        </ul>
                    </div>
                </section>
                <!-- //section03 -->

                <!-- section04 -->
                <section class="section04">
                    <div class="servie-wrap">
                        <h2 class="title"><span>TaaS : </span>Toy as a Service</h2>
                        <p class="sub-txt">"Toy와 Goods를 국내외 NFT 팀들이 손쉽게 <br class="mo">확보할 수 있는 서비스"</p>
                        <div class="service">
                            <ul class="service-group">
                                <li class="service-item item1">
                                    10년간<br class="pc">
                                    다양한 종류의 <br>
                                    <strong>아트토이를<br class="pc">
                                    자체 IP로 <br>
                                    직접 제작 및 판매</strong>
                                </li>
                                <li class="service-item item2">
                                    세계 최고 토이 기업 <br>
                                    <strong>팝마트와 전속 계약</strong> 및 <br>
                                    <strong>다양한 브랜드와 협업</strong><br>
                                    (나이키, 팝마트, <br class="pc">
                                    PUBG, <br class="pc">
                                    GXG 등)
                                </li>
                                <li class="service-item item3">
                                    다양한 내부 네트 워크, <br class="pc">
                                    글로벌 탑클래 <br>
                                    스토리 작가, <br class="pc">
                                    3D 모델러,<br>
                                    일러스트레이터,<br> 
                                    <strong>생산자 라인 구축</strong>
                                </li>
                                <li class="service-item item4">
                                    <strong>해외 전시를 통한 글로벌<br> 
                                    오프라인 인프라확보 : </strong><br>
                                    중국 을 시작으로 <br class="pc">
                                    세계 각국의 <br>
                                    아트 토이 <br class="pc">
                                    전시를 통한 <br class="pc">
                                    오프라인 <br>
                                    트래픽에 <br class="pc">
                                    브랜드 홍보 및<br class="pc"> 
                                    노출
                                </li>
                            </ul>
                        </div>

                        <div class="service-info">
                            <h3 class="title2">무형의 NFT를 유형의 실물로 제공</h3>
                            <p class="sub-txt">커뮤니티를 위한 아트토이 : 홀더들에게 <span>소속감 제공</span> + 다수 홀더들에게 <span>특별한 서비스 혜택</span></p>

                            <h3 class="title2">전 세계 탑클래스 품질</h3>
                            <p class="sub-txt">최고의 품질로 NFT의 고유한 개성 표현이 가능한 아트토이, 굿즈 생산가능
                            </p>
                            <p class="sub-txt2">TaaS는NFT생태계확장과함께보편화된개념이될것으로예상</p>
                        </div>

                        <h2 class="title"><span>TaaS : </span>진행절차</h2>
                        <div class="process-wrap">
                            <ul class="process">
                                <li class="process-item">
                                    <div class="process-num">01</div>
                                    <div class="process-img"><img src="./img/process-img1.png" alt=""></div>
                                    <h3 class="process-title">브랜드 선별 및 컨택</h3>
                                    <ul class="process-info">
                                        <li class="process-info-item">브랜드 협업 논의</li>
                                    </ul>
                                </li>
                                <li class="process-item">
                                    <div class="process-num">02</div>
                                    <div class="process-img"><img src="./img/process-img2.png" alt=""></div>
                                    <h3 class="process-title">브랜드 협업 및 TaaS 발표</h3>
                                    <ul class="process-info">
                                        <li class="process-info-item">브랜드 협업 범위 논의</li>
                                        <li class="process-info-item">TaaS 생산 범위 협의</li>
                                    </ul>
                                </li>
                                <li class="process-item">
                                    <div class="process-num">03</div>
                                    <div class="process-img"><img src="./img/process-img3.png" alt=""></div>
                                    <h3 class="process-title">TaaS 기획 및 설계</h3>
                                    <ul class="process-info">
                                        <li class="process-info-item">TaaS 제품 기획</li>
                                        <li class="process-info-item">제품 디자인</li>
                                        <li class="process-info-item">3D 제품 디자인인</li>
                                        <li class="process-info-item">TaaS 마케팅 진행</li>
                                    </ul>
                                </li>
                                <li class="process-item">
                                    <div class="process-num">04</div>
                                    <div class="process-img"><img src="./img/process-img4.png" alt=""></div>
                                    <h3 class="process-title">TaaS 제작</h3>
                                    <ul class="process-info">
                                        <li class="process-info-item">TaaS 샘플 제작</li>
                                        <li class="process-info-item">수정 및 보완</li>
                                        <li class="process-info-item">실제품 제작</li>
                                    </ul>
                                </li>
                                <li class="process-item">
                                    <div class="process-num">05</div>
                                    <div class="process-img"><img src="./img/process-img5.png" alt=""></div>
                                    <h3 class="process-title">TaaS 제품 출시</h3>
                                    <ul class="process-info">
                                        <li class="process-info-item">TaaS 제품 출시</li>
                                        <li class="process-info-item">브랜드 정책에 따라</li> 
                                        <li class="process-info-item">TaaS 운영</li>
                                    </ul>
                                </li>
                            </ul>
                        </div>
                    </div>
                </section>
                <!-- //section04 -->

                <!-- section05 -->
                <section class="section05">
                    <h2 class="title">Road Map</h2>
                    <div class="swiper loadmap-swiper">
                        <div class="swiper-button-prev"></div>
                        <div class="swiper-button-next"></div>
                        <div class="swiper-wrapper">
                            <div class="swiper-slide">
                                <h3 class="loadmap-title">IP & T.a.a.S</h3>
                                <ul class="loadmap">
                                    <li class="loadmap-item check"><i class="icon"></i>Taas 협업 진행 및 출시</li>
                                    <li class="loadmap-item check"><i class="icon"></i>NWT 실물 IP 아트토이 출시 (with POPMART)  </li>
                                    <li class="loadmap-item check"><i class="icon"></i>super B 브라운 브레스 출시</li>
                                    <li class="loadmap-item check"><i class="icon"></i>raBBiTT - pure b/w 출시</li>
                                    <li class="loadmap-item check"><i class="icon"></i>VANS 협업</li>
                                    <li class="loadmap-item check"><i class="icon"></i>ONE-PIECE 캐릭터 아트토이 제작</li>
                                    <li class="loadmap-item check"><i class="icon"></i>Popmart_ ENBT 블라인드박스</li>
                                    <li class="loadmap-item check"><i class="icon"></i>Popmart_ NBA 캐릭터 아트토이이 제작</li>
                                    <li class="loadmap-item check"><i class="icon"></i>한국 NIKE_FRP 제작</li>
                                    <li class="loadmap-item check"><i class="icon"></i>한국 아티스트 DEUX 아트토이, 굿즈 제작</li>
                                    <li class="loadmap-item check"><i class="icon"></i>PUBG 조형물/아트토이 제작</li>
                                    <li class="loadmap-item check"><i class="icon"></i>skycat 제작</li>
                                    <li class="loadmap-item check"><i class="icon"></i>캣보티카 TaaS 제작</li>
                                    <li class="loadmap-item check"><i class="icon"></i>skycat 아트토이 시리즈 유통</li>
                                    <li class="loadmap-item check"><i class="icon"></i>space HOOD 아트토이 시리즈 유통</li>
                                    <li class="loadmap-item check"><i class="icon"></i>타 기업 IP 확장</li>
                                    <li class="loadmap-item check"><i class="icon"></i>V2 래빗 캐릭터 런칭 (with POPMART)</li>
                                    <li class="loadmap-item check"><i class="icon"></i>V2 마우스 캐릭터 런칭 (with POPMART)</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>TaaS 제품 오프라인 런칭</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>김정기 작가 NFT 런칭</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>V0 츄츄 런칭</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>V1 츄츄,래빗,마우스 런칭</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>V3 츄츄 런칭</li>
                                </ul>
                            </div>
                            <div class="swiper-slide">
                                <h3 class="loadmap-title">UNIVERSE</h3>
                                <ul class="loadmap">
                                    <li class="loadmap-item check"><i class="icon"></i>2022년년 2월 - V0 민팅 완료</li>
                                    <li class="loadmap-item check"><i class="icon"></i>(클레이튼 최초 8k, Full3D 오브젝트 기반 아트 NFT)</li>
                                    <li class="loadmap-item check"><i class="icon"></i>TaaS개념 정립</li>
                                    <li class="loadmap-item check"><i class="icon"></i>V0 스테이킹 기능 도입 (V0포인트 생성)</li>
                                    <li class="loadmap-item ing"><i class="icon">ing</i>V1 펫효과 생성(합성시 포인트증가)</li>
                                    <li class="loadmap-item ing"><i class="icon">ing</i>V0 홀더 V1 에어드랍 혜택</li>
                                    <li class="loadmap-item ing"><i class="icon">ing</i>V2 Riding NFT 민팅</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>V3 IN GAME SKIN(캐릭터 스킨-no grade)</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>chu 코인 출시</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>V3  play to earn GAME 오픈</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>게임 개발</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>게임 IP 및 그래픽 구축</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>게임 내 획득 재화 토큰화</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>게임 대회 오픈(시즌제)</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>게임 내 파츠 구성</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>TaaS IP 민팅</li>
                                </ul>
                            </div>
                            <div class="swiper-slide">
                                <h3 class="loadmap-title">ECONOMY</h3>
                                <ul class="loadmap">
                                    <li class="loadmap-item check"><i class="icon"></i>핫토이, 퀸스튜디오 사 하이엔드 피규어어 제공</li>
                                    <li class="loadmap-item check"><i class="icon"></i>spaceHOOD 피규어 시리즈 제공</li>
                                    <li class="loadmap-item check"><i class="icon"></i>skycat 피규어 시리즈 제공</li>
                                    <li class="loadmap-item check"><i class="icon"></i>자사 IP 뱃지 시리즈 제공</li>
                                    <li class="loadmap-item check"><i class="icon"></i>NOICE 피규어 제공</li>
                                    <li class="loadmap-item check"><i class="icon"></i>티셔츠 제공</li>
                                    <li class="loadmap-item check"><i class="icon"></i>러그 제공</li>
                                    <li class="loadmap-item ing"><i class="icon">ing</i>온라인 포인트 사용처 제공</li>
                                    <li class="loadmap-item ing"><i class="icon">ing</i>홈페이지 개편</li>
                                    <li class="loadmap-item ing"><i class="icon">ing</i>V0 포인트 사용처</li>
                                    <li class="loadmap-item ing"><i class="icon">ing</i>V0 피규어 시리즈 제공</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>홀더 오프라인 사용처 구축</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>팝업스토어 구축</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>타 프로젝트 IP확장 및 메타버스 구축</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>온라인 소셜커머스 활용처 구축</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>V1 피규어 시리즈 제공</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>V2 피규어 시리즈 제공</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>V3 피규어 시리즈 제공</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>오프라인 포인트 사용처 제공</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>타프로젝트 홀더 PFP 아트토이화 제작</li>
                                    <li class="loadmap-item soon"><i class="icon">soon</i>타 기업 IP 확장</li>
                                </ul>
                            </div>
                        </div>
                    </div>
                </section>
                <!-- //section05 -->

                <!-- section06 -->
                <section class="section06">
                    <h2 class="title">Story</h2>
                    <div class="story">
                        <figure class="story-figure">
                            <img src="./img/story-img1.png" class="story-img" alt="">
                            <figcaption class="story-figcaption">chuchuCAT V.0</figcaption>
                        </figure>
                        <figure class="story-figure">
                            <img src="./img/story-img2.png" class="story-img" alt="">
                            <figcaption class="story-figcaption">chuchuCAT V.1</figcaption>
                        </figure>
                        <figure class="story-figure">
                            <img src="./img/story-img3.png" class="story-img" alt="">
                            <figcaption class="story-figcaption">chuchuCAT V.3</figcaption>
                        </figure>
                    </div>

                    <p class="stroy-txt">츄, 마우스, 래빗은 캐리어를 끌고다니며 꿈을 찾아 여행을 하는 꿈의 요정, 드디어 많은 꿈을 모았지만  마음은 허전했기에 각자 사랑을 찾아 또다시 여행을 하게되고 츄(별)을 모아 여정을 이어가는 도중 자신 들이 클론이라는 사실을 알게되어 함께 창조자를 향한 마지막 여행을 시작하게 되는데…</p>

                    <!-- footer -->
                    <footer id="footer">
                        <div class="footer-logo"><img src="./img/foot-logo.png" alt=""></div>
                        <div class="footer-copyright">NWT Copyright 2021~, All Right Reserved.</div>
                    </footer>
                    <!-- //footer -->
                </section>
                <!-- //section06 -->
            </div>
        </main>
        <!-- //container -->
    </div>

    <!-- 전체메뉴 -->
    <div id="sideMenu">
        <div class="menu-wrap">
            <div class="menu-logo"><img src="./img/logo.png" alt=""></div>
            <!-- MO 로그인 후 -->
            <div class="header-login-m">
                <img src="./img/login-img.png" alt=""  class="login-img">
                <div class="login-info">
                    <div class="login-id">0x43E8...0532</div>
                    <div class="login-point">총 가용 추<span>(포인트)</span> : 999,350</div>
                </div>
            </div>
            <!-- //MO 로그인 후 -->
            <ul class="menu-group">
                <li class="menu-item">
                    <a href="#" class="menu-link has-sub" aria-controls="sideProject">Project<i class="menu-link-icon"></i></a>
                    <ul id="sideProject" class="menu-depth2 collapse">
                        <li class="menu-depth2-item"><a href="#" id="team" class="menu-depth2-link">team</a></li>
                        <li class="menu-depth2-item"><a href="#" id="tass" class="menu-depth2-link">TaaS</a></li>
                    </ul>
                </li>
                <li class="menu-item"><a href="#" id="roadMap" class="menu-link">Road Map</a></li>
                <li class="menu-item"><a href="#" id="story" class="menu-link">Story</a></li>
                <li class="menu-item">
                    <a href="#" class="menu-link has-sub" aria-controls="sidePoint">Point<i class="menu-link-icon"></i></a>
                    <ul id="sidePoint" class="menu-depth2 collapse">
                        <li class="menu-depth2-item"><a href="#" class="menu-depth2-link">Check</a></li>
                        <li class="menu-depth2-item"><a href="#" class="menu-depth2-link">Histroy</a></li>
                    </ul>
                </li>
                <li class="menu-item">
                    <a href="#" class="menu-link has-sub" aria-controls="sideShop">Shop<i class="menu-link-icon"></i></a>
                    <ul id="sideShop" class="menu-depth2 collapse">
                        <li class="menu-depth2-item"><a href="#" class="menu-depth2-link">Goods</a></li>
                        <li class="menu-depth2-item"><a href="#" class="menu-depth2-link">Raffle</a></li>
                        <li class="menu-depth2-item"><a href="#" class="menu-depth2-link">Offine</a></li>
                    </ul>
                </li>
                <li class="menu-item">
                    <a href="#" class="menu-link has-sub" aria-controls="sidePlayground">Playground<i class="menu-link-icon"></i></a>
                    <ul id="sidePlayground" class="menu-depth2 collapse">
                        <li class="menu-depth2-item"><a href="#" class="menu-depth2-link">News</a></li>
                        <li class="menu-depth2-item"><a href="#" class="menu-depth2-link">Mini-Game</a></li>
                        <li class="menu-depth2-item"><a href="#" class="menu-depth2-link">CHul-Check</a></li>
                    </ul>
                </li>
            </ul>
        
            <ul class="menu-sns">
                <li class="discord"><a href="https://discord.com/invite/cye" target="_blank"><span class="a11y">디스코드</span></a></li>
                <li class="twitter">
                    <a href="https://twitter.com/closeyoureyesV0" target="_blank">
                        <span class="a11y">트위터</span>
                        <span class="sns-txt">V0</span>
                </a>
                </li>
                <li class="twitter">
                    <a href="https://twitter.com/CYE_nft" target="_blank">
                        <span class="a11y">트위터</span>
                        <span class="sns-txt">V1</span>
                    </a>
                </li>
                <li class="telegram"><a href="" target="_blank"><span class="a11y">텔레그램</span></a></li>
            </ul>
        
            <ul class="menu-lang">
                <li class="menu-lang-item"><a href="#" target="_blank" class="menu-lang-link" aria-label="국문 사이트 바로가기">Kor</a></li>
                <li class="menu-lang-item"><a href="#" target="_blank" class="menu-lang-link eng" aria-label="영문 사이트 바로가기">Eng</a></li>
            </ul>

            <div class="btn-wrap menu-logout">
                <button type="button" class="btn btn-logout">Logout</button>
            </div>

            <button type="button" aria-label="전체 메뉴닫기" class="btn-menu-closed" onclick="popClose('#sideMenu')"></button>
        </div>
        <div class="menu-dim" onclick="popClose('#sideMenu')"></div>
    </div>
    <!-- //전체메뉴 -->
   
   <script>
        window.addEventListener('DOMContentLoaded', function() {
            docSlider.init();

            var loadmapSwiper = undefined;
            function initLoadmapSwiper() {
                var screenWidth = window.innerWidth;
                if(screenWidth > 980 && loadmapSwiper === undefined) {
                    loadmapSwiper = new Swiper('.loadmap-swiper',{
                        loop: true,
                        navigation: {
                            nextEl: ".swiper-button-next",
                            prevEl: ".swiper-button-prev",
                        },
                        slidesPerView: 'auto',
                        centeredSlides: true,
                        on: {
                            init: function() {
                                this.slides.forEach(function(ele) {
                                    ele.setAttribute('tabindex', 0)
                                });
                            }
                        },
                    });
                } else if(screenWidth < 981 && loadmapSwiper !== undefined) {
                    loadmapSwiper.destroy();
                    loadmapSwiper = undefined;
                    document.querySelector('.loadmap-swiper .swiper-wrapper').removeAttribute('style');
                    document.querySelector('.loadmap-swiper .swiper-slide').removeAttribute('style');
                }
            }
            initLoadmapSwiper();
            window.addEventListener('resize', function() {
                initLoadmapSwiper();
            })
        });

        document.querySelector('#team').addEventListener('click', function(e) {
            document.querySelector('[data-ds-jump="2"]').click();
            popClose('#sideMenu');
        })

        document.querySelector('#tass').addEventListener('click', function(e) {
            document.querySelector('[data-ds-jump="3"]').click();
            popClose('#sideMenu');
        })

        document.querySelector('#roadMap').addEventListener('click', function(e) {
            document.querySelector('[data-ds-jump="4"]').click();
            popClose('#sideMenu');
        })

        document.querySelector('#story').addEventListener('click', function(e) {
            document.querySelector('[data-ds-jump="5"]').click();
            popClose('#sideMenu');
        })
    </script>
</body>
</html>