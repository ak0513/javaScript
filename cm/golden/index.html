<!DOCTYPE html>
<html lang="kr">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="refresh" content="2;url=https://open.kakao.com/o/sUlUnkYe">
    <title>골든모바일</title>
</head>
<body>
    <style>
        body,html {margin:0;padding:0;}
        #wrap {margin:0;padding:0;text-align:center;}
        #wrap img {max-width:100%;height:auto;}
    </style>
    <div id="wrap">
        <img src="./img/golden1.jpg" alt="폰테크 1등 골든 모바일">
        <br><br>
        <a href="https://open.kakao.com/o/sUlUnkYe"><img src="./img/golden2.jpg" alt="골든모바일 카카오톡 바로가기"></a>
    </div>
</body>
</html>